bin/cqlsh.py

```diff
@@ -870,7 +870,7 @@ def use_stdin_reader(self, until='', prompt=''):
                 return
             yield newline
 
-    def cmdloop(self):
+    def cmdloop(self, intro=None):
```

Comment:
I don't see this var being used within this function, so why is this needed? Am I just blind somehow?

Reply:
I dont see where it is used either.

Reply:
Ah, that makes sense. Optionally you could add a comment that says:

```python
def cmdloop(self, intro=None):  # intro is unused, but included to match signature of parent `Cmd.cmdloop()`
```

As otherwise I could see someone in the future changing this... actually, the best way to guard against that is get this merged and fix any other `pylint` issues, then enable `pylint` in CI so that it prevents stuff like this creeping back in...